integrations/kosli/manifest.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "Kosli",
+  "version": "1.0.0",
+  "overview": "Track changes to your feature flags in a Kosli Trail.",
+  "description": "Kosli records all of the changes in your software to give you the easy buttons for audit, compliance, security, and incident response. Integrate LaunchDarkly with Kosli to see your feature flag changes for each environments in a separate Kosli Trail.",
+  "details": "Turn on the LaunchDarkly integration within the Kosli app (https://app.kosli.com/) to get a webhook and secret to begin reporting changes to your flags in a Kosli trail.",
+  "author": "Kosli",
+  "supportEmail": "support@kosli.com",
+  "links": {
+    "site": "https://www.kosli.com",
+    "privacyPolicy": "https://www.kosli.com/privacy-policy/",
+    "supportWebsite": "https://www.kosli.com/community/"
+  },
+  "categories": ["developer-tools", "monitoring"],
+  "icons": {
+    "square": "assets/images/square.svg",
+    "horizontal": "assets/images/horizontal.svg"
+  },
+  "otherCapabilities": ["external"],
+  "formVariables": [
+    {
+      "key": "webhookUrl",
+      "name": "Kosli Webhook URL",
+      "type": "string",
+      "description": "Enter your Kosli webhook URL"
+    },
+    {
+      "key": "secret",
+      "name": "Kosli secret",
+      "type": "string",
+      "description": "Enter your Kosli secret",
+      "isSecret": true
+    }
+  ],
+  "capabilities": {
+    "auditLogEventsHook": {
+      "endpoint": {
+        "url": "{{webhookUrl}}",
+        "method": "POST",
+        "headers": [
+          {
+            "name": "x-ld-signature",
+            "value": "{{secret}}"
+          }
+        ]
+      },
+      "templates": {
```

Comment:
Can you make `templates` and empty object `"templates": {},` and then you can remove the `templates` directory. You have `useStandardWebhookPayload` set to `true` below so no template should be needed.

Reply:
Thank you, I've updated the manifest with these changes.